### ISSUE TITLE
Possible bug: pg_fisher_loss = -action_log_probs.mean()

### ISSUE THREAD

**Issue by mingfeisun** [ikostrikov/pytorch-a2c-ppo-acktr#304]
Hi Ilya, 

Thanks for the good implementation of Acktr in pytorch. I'm not sure about the correctness of the following line: 

https://github.com/ikostrikov/pytorch-a2c-ppo-acktr-gail/blob/41332b78dfb50321c29bade65f9d244387f68a60/a2c_ppo_acktr/algo/a2c_acktr.py#L56

Shouldn't it be: `pg_fisher_loss = action_log_probs.mean() `? Then the total loss would make sense as it'd represent the joint likelihood. 

Also, according to https://github.com/openai/baselines/blob/master/baselines/acktr/acktr.py#L48, the baseline implementation has enabled neglogpac so a minus is needed. but in your implementation a minus may not be needed.
